src/MediaRatingTracker.java

```diff
@@ -0,0 +1,76 @@
+import components.map.Map1L;
+
+/**
+ * Component to track ratings for movies and shows.
+ *
+ * @param <K>
+ *            the name of the movie or show
+ * @param <V>
+ *            the rating
+ * @author Wes Waterstreet
+ *
+ */
+public class MediaRatingTracker<K, V> extends Map1L<K, V> {
```

Comment:
I wouldn't extend an existing component in your actually design. Either you use a map or you are a map, not both.